templates/inspec/singular_resource.erb

```diff
@@ -42,14 +42,14 @@ class <%= object.__product.name.camelize(:upper) -%><%= object.name -%> < GcpRes
   def initialize(params)
     super(params.merge({ use_http_transport: true }))
     @params = params
-    @fetched = @connection.fetch(product_url, resource_base_url, params, 'Get')
+    @fetched = @connection.fetch(product_url(params[:beta]), resource_base_url, params, 'Get')
```

Comment:
What's the purpose of params[:beta]? Just a way for users to explicitly set if they want to use the beta APIs or not?

Reply:
Exactly